src/node/install.sh

```diff
@@ -232,7 +232,7 @@ else
         [ ! -z "$https_proxy" ] && npm set https-proxy="$https_proxy"
         [ ! -z "$no_proxy" ] && npm set noproxy="$no_proxy"
         npm install -g pnpm
-        pnpm setup
+        ENV="$HOME/.bashrc" SHELL="$(which bash)" pnpm setup
```

Comment:
I am not confident if `$HOME` would resolve correctly. Hence, I think you need to compute Home Dir as similar to https://github.com/devcontainers/features/blob/65fb90b7fc8634986ab5644d00d66b983e9a50df/src/common-utils/main.sh#L418-L429